contracts/ERC677BridgeToken.sol

```diff
@@ -49,14 +51,19 @@ contract ERC677BridgeToken is
     }
 
     function getTokenInterfacesVersion() public pure returns(uint64 major, uint64 minor, uint64 patch) {
-        return (2, 0, 0);
+        return (3, 0, 0);
```

Comment:
The changes I made are not essentional but do change the exposed API, so by semver this requests for major change